src/main/kotlin/org/opensearch/commons/notifications/NotificationConstants.kt

```diff
@@ -70,9 +69,5 @@ object NotificationConstants {
     const val ALLOWED_CONFIG_FEATURE_LIST_TAG = "allowed_config_feature_list"
     const val PLUGIN_FEATURES_TAG = "plugin_features"
```

Comment:
Also these other mentions of FEATURE? Are they still used?

Reply:
These all fall under the `"There are still some indirect references to features like the enabled features returned in the GetPluginFeatures API which will be addressed in a follow-up PR."` category.

I'll remove that then since that API is likely going to be taken out.